test/parallel/test-timers-active-handles/deasync.cc

```diff
@@ -0,0 +1,18 @@
+#include <node.h>
+#include <uv.h>
```

Comment:
wrong place for this test. All these files would go in `test/addon/<test-name/`. If you are going to include any module then the `node_module` folder needs to be checked in with the test. Though we generally don't allow third party dependencies in our tests, more so in native tests. Because native APIs break faster than module authors can keep up. And we aren't in the habit of floating patches on top of other libraries.

IMO this test will need to be written stand alone.